internal/configsource/source.go

```diff
@@ -198,36 +174,23 @@ func BuildConfigSources(ctx context.Context, configSourcesSettings map[string]Se
 //
 //	component:
 //	  # Retrieves the value from the file text.txt located on the path specified by the environment
-//	  # variable DATA_PATH. The name of the environment variable is the string after the delimiter
-//	  # until the first character different than '_' and non-alpha-numeric.
-//	  text_from_file: $file:$DATA_PATH/text.txt
-//
-// Since environment variables and config sources both use the '$', with or without brackets, as a prefix
-// for their expansion it is necessary to have a way to distinguish between them. For the non-bracketed
-// syntax the code will peek at the first character other than alpha-numeric and '_' after the '$'. If
-// that character is a ':' it will treat it as a config source and as environment variable otherwise.
-// For example:
-//
-//	component:
-//	  field_0: $PATH:/etc/logs # Injects the data from a config sourced named "PATH" using the selector "/etc/logs".
-//	  field_1: $PATH/etc/logs  # Expands the environment variable "PATH" and adds the suffix "/etc/logs" to it.
+//	  # variable DATA_PATH.
+//	  text_from_file: ${file:${DATA_PATH}/text.txt}
```

Comment:
```suggestion
//	  text_from_file: ${file:${env:DATA_PATH}/text.txt}
```
do we need to explicitly add `env` in such examples?